test/e2e/multiarch-container/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -x
+VERSIONS="0.1 0.2 0.3"
```

Comment:
What is the purpose of the versions? Could they all just be 0.1?

Reply:
I'm still confused, why does the tag matter? Why not just change all the tests to use the same 0.1 tag?

Reply:
I guess you're right, there is no test case to update an image with a sync or a rollback. I was for some reason just assuming there would be one. And I believe there _should_ be one.

However, I think having these three image versions for any future test are good. They build and push in seconds, so not much overhead there.